supertokens_python/recipe/session/framework/fastapi/__init__.py

```diff
@@ -32,12 +33,15 @@ def verify_session(
     ] = None,
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[..., Coroutine[Any, Any, Union[SessionContainer, None]]]:
-    if user_context is None:
-        user_context = {}
+    _ = user_context
```

Comment:
why is this needed?

Reply:
Usage of user_context is 2nd order nested and somehow linter say it's not used. So they throw an error.